3-datamodels/README.rst

```diff
@@ -237,16 +237,16 @@ Now we will ask for records matching the query "refer". Go to:
 
 http://192.168.50.10/api/custom_records/?q=refer
 
-The second record is still return even though it does not contain the word
+The second record is still returned even though it does not contain the word
 "refer". This is possible because we asked our search engine to analyze the
 text as "english".
 
 Open the file:
 
 **custom_data_module/mappings/custom_record/custom-record-v1.0.0.json**
 
-This is the Elasticearch Mapping which enables to tell our search engine
-how to analyze the text. You can go see Elasticsearch documentation
+This is the Elasticsearch **Mapping** file. It tells our search engine
```

Comment:
Hmm, maybe highlight also "Elasticsearch mapping" as a phrase together?  (because otherwise mapping is a too generic word perhaps...)